src/ralph/lib/transitions/api/views.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+from django import forms
+from django.core.urlresolvers import reverse
+from rest_framework import serializers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from ralph.api import RalphReadOnlyAPIViewSet
+from ralph.api.fields import ModelMultipleChoiceField
+from ralph.lib.transitions.api.serializers import (
+    TransitionActionSerializer,
+    TransitionJobSerializer,
+    TransitionModelSerializer,
+    TransitionSerializer
+)
+from ralph.lib.transitions.models import (
+    Action,
+    run_transition,
+    Transition,
+    TransitionJob,
+    TransitionModel
+)
+from ralph.lib.transitions.views import collect_actions
+
+FIELD_MAP = {
+    forms.CharField: (serializers.CharField, [
+        'max_length', 'initial', 'required'
+    ]),
+    forms.BooleanField: (serializers.BooleanField, ['initial', 'required']),
+    forms.URLField: (serializers.URLField, ['initial', 'required']),
+    forms.IntegerField: (serializers.IntegerField, ['initial', 'required']),
+    forms.DecimalField: (serializers.DecimalField, ['initial', 'required']),
+    forms.DateField: (serializers.DateField, ['initial', 'required']),
+    forms.DateTimeField: (serializers.DateTimeField, ['initial', 'required']),
+    forms.TimeField: (serializers.TimeField, ['initial', 'required']),
+    forms.ModelMultipleChoiceField: (ModelMultipleChoiceField, [
+        'initial', 'required', 'choices'
+    ]),
+    forms.ModelChoiceField: (serializers.ChoiceField, [
+        'initial', 'required', 'choices'
+    ]),
+    forms.ChoiceField: (serializers.ChoiceField, [
+        'initial', 'required', 'choices'
+    ]),
+}
+
+
+class TransitionJobViewSet(RalphReadOnlyAPIViewSet):
+    queryset = TransitionJob.objects.all()
+    serializer_class = TransitionJobSerializer
+
+
+class TransitionModelViewSet(RalphReadOnlyAPIViewSet):
+    queryset = TransitionModel.objects.all()
+    serializer_class = TransitionModelSerializer
+
+
+class TransitionActionViewSet(RalphReadOnlyAPIViewSet):
+    queryset = Action.objects.all()
+    serializer_class = TransitionActionSerializer
+
+
+class TransitionViewSet(RalphReadOnlyAPIViewSet):
+    queryset = Transition.objects.all()
+    serializer_class = TransitionSerializer
+    prefetch_related = ['actions']
+
+
+class TransitionView(APIView):
+
+    def dispatch(self, *args, **kwargs):
```

Comment:
why `obj_pk` and `transition_id` (btw - pk and id - can't we do this uniformly?) are not explicit params here?